WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -49,8 +49,7 @@ public class WPScreenshotTest extends BaseTest {
 
 
     private DemoModeEnabler mDemoModeEnabler = new DemoModeEnabler();
-
-    @Ignore("Ignored until there's a way to exclude tests on FTL properly, via `--test-targets` option.")
+    
```

Comment:
Note that commit 789eb204a533e2f778b4349467045614b4ebbf1a that changed this line in https://github.com/wordpress-mobile/release-toolkit/pull/403 and that we reverted in this PR… still had the `@Ignore` annotation already (just without an explanation message until https://github.com/wordpress-mobile/release-toolkit/pull/403).

I'm not sure why the test was already `@Ignore`'d even before we had to apply the workaround for the last of `test_target:` parameter in our FTL action. Probably because the `WPScreenshotTest` was in a bad place and failing at that time so someone decided to ignore it (and forgot to document why)?

But since the whole goal of this PR is to be in a good place to be able to work on Android screenshot tests again, so without them being `@Ignore`'d when we need to run them explicitly and in isolation, I figured it made more sense to remove the `@Ignore` completely here, instead of _just_ reverting https://github.com/wordpress-mobile/release-toolkit/pull/403 and keeping the `@Ignore` that we had before the workaround.

Reply:
> I'm not sure why the test was already @Ignore'd even before we had to apply the workaround for the last of test_target: parameter in our FTL action. Probably because the WPScreenshotTest was in a bad place and failing at that time so someone decided to ignore it

Exactly, the `WPScreenshotTest` was already ignored before #16970 because it was failing. #16970 really only disables the JP one, but adds a comment for both tests.